brave-people.json

```
@@ -429,5 +429,20 @@
   "y" : 12,
   "color" : "#000FFF",
   "username" : "zakcseen"
+}, {  
+  "x" : 8,
+  "y" : 0,
+  "color" : "#000FFF",
+  "username" : "Oppus"
+}, {  
+  "x" : 19,
+  "y" : 1,
+  "color" : "#00CBD4",
+  "username" : "vlzz1221"
+}, {  
+  "x" : 2,
+  "y" : 16,
+  "color" : "#00CCFF",
+  "username" : "xenophane"
```

```suggestion
}, {  
  "x" : 2,
  "y" : 16,
  "color" : "#00CCFF",
  "username" : "xenophane"
```

请不要添加额外的内容。

另外你有没有发现你吧自己的ID拼写错了？ `xenophane` -> `xenphane`